packages/main/src/List.js

```diff
@@ -371,6 +421,39 @@ class List extends UI5Element {
 		this.addEventListener("ui5-_forward-before", this.onForwardBefore.bind(this));
 		this.addEventListener("ui5-_selection-requested", this.onSelectionRequested.bind(this));
 		this.addEventListener("ui5-_focus-requested", this.focusUploadCollectionItem.bind(this));
+
+		this._handleResize = this.checkListInViewport.bind(this);
+		this.i18nBundle = getI18nBundle("@ui5/webcomponents");
+	}
+
+	onExitDOM() {
+		this.unobserveListEnd();
+		this.resizeListenerAttached = false;
+		ResizeHandler.deregister(this.getDomRef(), this._handleResize);
+	}
+
+	onBeforeRendering() {
+		this.prepareListItems();
+	}
+
+	onAfterRendering() {
+		if (this.growsOnScroll) {
+			this.observeListEnd();
+		} else if (this.listEndObserved) {
+			this.unobserveListEnd();
+		}
+
+		if (this.grows) {
```

Comment:
this.grows or rather this.growsOnScroll? Do we need the resize listener for button mode?

Reply:
Maybe we do for the busy indicator. I think I got it